MAUI/MauiAppBasic/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,70 @@
+{
+  "Sample": {
```

Comment:
>Sample [](http://example.com/codeflow?start=3&length=6)

Finish the metadata section , see example - [https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/master/4-WebApp-your-API/4-1-MyOrg/AppCreationScripts/sample.json](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/master/4-WebApp-your-API/4-1-MyOrg/AppCreationScripts/sample.json). 
Also see Sample Development Guidance Template.docx for more info.

Reply:
active-directory-xamarin-native-v2